File: src/main/java/um/haberes/report/model/haberes/core/CursoDto.java
package um.haberes.report.model.haberes.core;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CursoDto {

    private Long cursoId;

    private String nombre = "";

    private Integer facultadId;

    private Integer geograficaId;

    private byte anual = 0;

    private byte semestre1 = 0;

    private byte semestre2 = 0;

    private Integer nivelId;

    private byte adicionalCargaHoraria = 0;

    private FacultadDto facultad;

    private GeograficaDto geografica;

    private NivelDto nivel;

}

File: src/main/java/um/haberes/report/kotlin/dto/CursoDto.kt
package um.haberes.report.kotlin.dto

data class CursoDto(

    var cursoId: Long? = null,

    var nombre: String = "",
    var facultadId: Int? = null,
    var geograficaId: Int? = null,
    var anual: Byte = 0,
    var semestre1: Byte = 0,
    var semestre2: Byte = 0,
    var nivelId: Int? = null,
    var adicionalCargaHoraria: Byte = 0,
    var facultad: FacultadDto? = null,
    var geografica: GeograficaDto? = null,
    var nivel: NivelDto? = null

)
